PublicDashboards: Add setting to disable the feature (#78894)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

Move an individual app plugin page (referenced by its `path` field) to a specific navigation section.
Format: `<pageUrl> = <sectionId> <sortWeight>`

## [public_dashboards]

This section configures the [public dashboards]({{< relref "../../dashboards/dashboard-public" >}}) feature.

### enabled

> Set this to `false` to disable the public dashboards feature. This will prevent users from creating new public dashboards and disable all existing ones.

Reviewer: ```suggestion
Set this to `false` to disable the public dashboards feature. This prevents users from creating new public dashboards and disabling existing ones.
```

Author: This doesn't have the same meaning but I can change it to something like:
```suggestion
Set this to `false` to disable the public dashboards feature. This prevents users from creating new public dashboards and disables existing ones.
```

Reviewer: good catch, I missed needing a change to 'disables'. Thank you.
